btree/btree.c

```diff
@@ -1161,10 +1238,11 @@ static void node_refcnt_update(struct nd *node, bool increment);
 #ifndef __KERNEL__
 static void node_set_level  (const struct nd *node, uint8_t new_level,
 			     struct m0_be_tx *tx);
-static void node_move (struct nd *src, struct nd *tgt,
-		       enum dir dir, int nr, struct m0_be_tx *tx);
+static void node_move (struct nd *src, struct nd *tgt, enum dir dir, int nr,
```

Comment:
variable name and enum dir name are same. May be we can have different variable name.